R/tags.R

```diff
@@ -292,7 +292,14 @@ tagFunction <- function(func) {
 #' @export
 tagAppendAttributes <- function(tag, ...) {
   throw_if_tag_function(tag)
-  tag$attribs <- c(tag$attribs, dropNullsOrEmpty(dots_list(...)))
+  newAttribs <- dropNullsOrEmpty(dots_list(...))
+  if (any(names2(newAttribs) == "")) {
```

Comment:
Just to be consistent

```suggestion
  if (any(!nzchar(names2(newAttribs)))) {
```